paynow.php

```diff
@@ -174,7 +174,9 @@ private function createModuleSettings()
             Configuration::updateValue('PAYNOW_PAYMENT_VALIDITY_TIME', 86400) &&
             Configuration::updateValue('PAYNOW_ORDER_ABANDONED_STATE', Configuration::get('PAYNOW_ORDER_INITIAL_STATE')) &&
             Configuration::updateValue('PAYNOW_ORDER_EXPIRED_STATE', Configuration::get('PAYNOW_ORDER_INITIAL_STATE')) &&
-            Configuration::updateValue('PAYNOW_CREATE_ORDER_STATE', 1);
+            Configuration::updateValue('PAYNOW_CREATE_ORDER_STATE', 1) &&
+            Configuration::updateValue('PAYNOW_RETRY_PAYMENT_BUTTON_ENABLED', 1) &&
+            Configuration::updateValue('PAYNOW_RETRY_BUTTON_ORDER_STATE', join(',', [8,20,12]));
```

Comment:
czy nie powinniście wyciągnąć odpowiednich ID z bazy? Zakładam że każdy sklep może mieć inne ID.

Reply:
To są ID-ki domyślnych statusów z bazowej instalacji PS, tutaj je przyjmujemy jako wartości startowe konfiguracji.  Każdy skep te bazowe powinien mieć, można oczywiści dokładać nowe / swoje statusy (np. jakaś wtyczka może dodać). Tutaj zakładam, że jeśli developer w sklepie zmodyfikował oryginalne flow statusów, to wówczas może z poziomu konfiguracji wtyczki Paynow zmienić wybór domyśłny.

Reply:
ok, widziałem już instalacje gdzie ktoś usuwał wszystkie statusy i była zupełnie inna kolejność